app/controllers/cart_items_controller.rb

```diff
@@ -6,7 +6,7 @@ def create
 		item = Item.find(params[:item_id])
 		@cart.add_item(item.id)
 		session[:cart] = @cart.contents
-		flash[:notice] = "You have #{pluralize(@cart.count_of, "item")} in your cart."
+		flash[:notice] = {color: "white", message: "You have #{pluralize(@cart.count_of, "item")} in your cart."}
```

Comment:
Line is too long. [107/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer single-quoted strings inside interpolations.<br>Tab detected.